remake.yml

```diff
@@ -34,20 +34,21 @@ targets:
     command: download_nwis_site_data(target_name)
     depends: fetch_date
     
-  all_site_files:
-    command: hash_dir_files('1_fetch/out/site_data/01466500.csv',
+  1_fetch/out/all_site_files.rds:
+    command: hash_dir_files(out_file = target_name,
```

Comment:
picky, but why not `hash_files` instead of `hash_dir_files`?
Double picky, but scipiper has a built in function for this, called `combine_to_ind` (which is a wrapper for `sc_indicate` but fits the function_name(target_name, file1, file2, file3,...) pattern)

Using the built-in function can help avoid issues with R versions.

Reply:
`dir` because originally this function was taking a directory name, makes less sense now though. 

 I thought there ought to be a built-in way to do it, I didn't look for the right terms though apparently